Bottom: d1354e7b9d1b968ca9b674b3fd02b5d2f914ee94
Top:    faa1317e1251b40eab14c2d4bd0b8c739c26e6da
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-07 17:38:03 +0800

Refresh of debug.patch

---

diff --git a/lib/cache.c b/lib/cache.c
index 7a2caa6..23462a2 100644
--- a/lib/cache.c
+++ b/lib/cache.c
@@ -658,6 +658,8 @@ int cache_update_offset(struct mapent_cache *mc, const char *mkey, const char *k
 		return CHE_FAIL;
 
 	me = cache_lookup_distinct(mc, key);
+	if (me)
+		error(LOGOPT_ANY, "owner %p owner->key %s me %p me->key %s", owner,owner->key,me,me->key);
 	if (me && me->age == age) {
 		if (me == owner)
 			return CHE_DUPLICATE;
